L1Trigger/L1THGCal/interface/hw_interfaces/HGCalHardwareInterface.h

```diff
@@ -0,0 +1,59 @@
+#ifndef __L1Trigger_L1THGCal_HGCalHardwareInterface_h__
+#define __L1Trigger_L1THGCal_HGCalHardwareInterface_h__
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/L1THGCal/interface/HGCalCluster.h"
+#include "DataFormats/L1THGCal/interface/HGCalMulticluster.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+class HGCalHardwareInterface{
+  
+ public:
+  /* mapping is retrieved from: https://twiki.cern.ch/twiki/bin/viewauth/CMS/HGCALTriggerArchitectureDesignAndImplementation */
+  struct ClusterDataFormat{
+    enum CDF {ET = 0, CenterY = 8, CenterX = 20, Flags = 32, SizeY = 38, SizeX = 46, LocalMax = 54, NoTriggerCells = 56, LocalMax1RelX = 64,  LocalMax0ET = 72, LocalMax0RelY = 80, LocalMax0RelX = 88, LocalMax2RelY = 96,  LocalMax2RelX = 104,  LocalMax1ET = 112,  LocalMax1RelY = 120,  LocalMax3ET = 128, LocalMax3RelY = 136,  LocalMax3RelX = 144,  LocalMax2ET = 152 };
+  };
+  struct MulticlusterDataFormat{
```

Comment:
Hi, I think we should have two classifications: "Cluster steps" and "cluster types" 
These data mapping describe the container information from the first layer FPGA to the second layer FPGA, whereas the Cluster2D and Cluster3D are the types of clusters. Please see my next commit to see the enumerated types.